### ISSUE TITLE
J key needs to be pressed twice in order to work

### ISSUE THREAD

**Issue by Koaieus** [Athou/commafeed#503]
While viewing any feed or category or all, after about three items the J key needs to be pressed twice in order to open the next item. It seems like the first press moves the screen to the next item, while the second press subsequently selects that item (it puts a blue line next to the article) without moving the screen. I use expanded view. The similar problem goes for pressing the K key after going down some items using J: it moves the screen back up 2 items, but places the selection only one item back. This can especially be confusing when the articles are long and about as tall as the screen. After the first press on K, it works normally.

tl;dr:
Before (selected in curly):
{1} 
[2]
[3]
Press J:
[2]
[3]
[4]
OPTION A: Press J again:
{2}  !!! screen does not move
[3]
[4]
OPTION B: Press K:
[0] !!! screen moves too far back
{1}
[2]
